packages/cli/src/utils/logger.ts

```diff
@@ -0,0 +1,79 @@
+export class Logger {
+  private _isVerbose = false;
+  private _isDebug = false;
+  private _isLogEnabled = true;
+
+  public setIsVerbose(enable: boolean): this {
```

Comment:
Don't use `_` for private properties (it's used in JS as a convention as there's no private properties there), or `public` to match code conventions in the repo.